reactive/jenkins.py

```diff
@@ -230,3 +248,34 @@ def update_nrpe_config(nagios):
 def stop():
     service_stop("jenkins")
     status_set("maintenance", "Jenkins stopped")
+
+
+@when("reset.jenkins.storage")
+def reset_jenkins_storage():
```

Comment:
Perhaps I'm missing something but I'm not sure how the unit tests will work with the remove_state and get_state calls and regardless since those are reactive states they seem like they should be in reactive/jenkins.py